Sources/WalletConnectPairing/PairingClient.swift

```diff
@@ -3,29 +3,60 @@ import WalletConnectUtils
 import WalletConnectRelay
 import WalletConnectNetworking
 import Combine
+import JSONRPC
 
 public class PairingClient: PairingRegisterer {
+
+    public var pingResponsePublisher: AnyPublisher<(String), Never> {
+        pingResponsePublisherSubject.eraseToAnyPublisher()
+    }
+    public let socketConnectionStatusPublisher: AnyPublisher<SocketConnectionStatus, Never>
```

Comment:
cannot find, why we need it here?

Reply:
which clients in your opinion should expose `socketConnectionStatusPublisher`?